inspirehep/modules/forms/static/js/forms/inspire-form.js

```diff
@@ -468,12 +468,14 @@ define(function(require, exports, module) {
 
       set_status(tpl_status_saving());
 
-      $.ajax(
+      var promise = $.ajax(
```

Comment:
And I separated it because someone on the internet was wrong and was blaming the issue on Firefox to this chaining.